basicLibrary/src/main/java/basicLibrary/lab3.java

```diff
@@ -0,0 +1,64 @@
+package basicLibrary;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class lab3 {
+    public String analyzeWeather(int[][] arr){
+        int count = 1;
+        int max = 0;
+        int min = 1000;
```

Comment:
Instead of hard coding, it's also common to initialize variables like this with the first value in the data structure you're working with (the way you did it is perfectly fine though!):
MAX = arr[0][0];